src/client/scss/utils/_grid-framework.scss

```diff
@@ -0,0 +1,81 @@
+// Framework grid generation
+//
+// Used only by Bootstrap to generate the correct number of grid classes given
+// any value of `$grid-columns`.
+
+// [converter] This is defined recursively in LESS, but Sass supports real loops
+@mixin make-grid-columns($i: 1, $list: ".col-xs-#{$i}, .col-sm-#{$i}, .col-md-#{$i}, .col-lg-#{$i}") {
```

Comment:
@vitkuz that's the bootsrap code, right? why do you need to commit if you can get it from dependencies?

Reply:
i just use it as my mixin )